charts/device-plugin-operator/values.yaml

```diff
@@ -1,4 +1,17 @@
-image:
-  hub: intel
-  tag: ""
-  pullPolicy: IfNotPresent
\ No newline at end of file
+manager:
+  image:
+    hub: intel
+    tag: ""
+    pullPolicy: IfNotPresent
+
+kubeRbacProxy:
+  image:
+    hub: ghcr.io
+    hubRepo: kubebuilder
+    tag: v0.14.1
+    pullPolicy: IfNotPresent
+
+privateRegistry:
+  registryUrl: ""
+  registryUser: ""
+  registrySecret: ""
```

Comment:
For these, you'd also need to include the secret creation. I'd be fine to leave it as a later improvement.